models/iaf_psc_exp.cpp

```diff
@@ -248,8 +248,6 @@ nest::iaf_psc_exp::calibrate()
 
   const double h = Time::get_resolution().get_ms();
 
-  // numbering of state vaiables: i_0 = 0, i_syn_ = 1, V_m_ = 2
-
   // commented out propagators: forward Euler
```

Comment:
The Euler propagators should go. If at all, I would add a remark to the model documentation pointing out that this neuron model differs from the implemented in [1]_ by using exact integration instead of forward Euler.

Reply:
The comment says:
```
  // commented out propagators: forward Euler
  // needed to exactly reproduce Tsodyks network
```
Perhaps it makes sense for people coming after us to leave this in? We could also wrap it in some preprocessor conditions.

Reply:
I am strictly against leaving commented-out code in. Anyone with a minimal understanding of Euler's method should be able to re-Eulerize the code if they want to (we should put a remark into the model doc). Compiler switches are too cumbersome, since we'd need to propagate them from the CMake level. The only alternative I would consider would be a runtime switch in the model, similar to the `consistent_integration` switch in the Izhikevich model.